components/base/sensorcontrolled/velocities.go

```diff
@@ -173,21 +173,21 @@ func sign(x float64) float64 { // A quick helper function
 func (sb *sensorBase) SetState(ctx context.Context, state []*control.Signal) error {
 	sb.mu.Lock()
 	defer sb.mu.Unlock()
-	if sb.isPolling() {
-		// if the spin loop is polling, don't call set velocity, immediately return
-		// this allows us to keep the control loop running without stopping it until
-		// the resource Close has been called
-		sb.logger.CInfo(ctx, "skipping set state call")
-		return nil
-	}
+	// if sb.isPolling() {
+	// 	// if the spin loop is polling, don't call set velocity, immediately return
+	// 	// this allows us to keep the control loop running without stopping it until
+	// 	// the resource Close has been called
+	// 	sb.logger.CInfo(ctx, "skipping set state call")
+	// 	return nil
+	// }
 
 	sb.logger.CDebug(ctx, "setting state")
 	linvel := state[0].GetSignalValueAt(0)
 	// multiply by the direction of the linear velocity so that angular direction
 	// (cw/ccw) doesn't switch when the base is moving backwards
 	angvel := (state[1].GetSignalValueAt(0) * sign(linvel))
 
-	return sb.SetPower(ctx, r3.Vector{Y: linvel}, r3.Vector{Z: angvel}, nil)
+	return sb.controlledBase.SetPower(ctx, r3.Vector{Y: linvel}, r3.Vector{Z: angvel}, nil)
```

Comment:
switched this to using the underlying SetPower function so the spin does not cancel its own polling

Reply:
@martha-johnston maybe this will help in motor_encoder's SetSate as well, just calling the underlying motor.

Reply:
Thus bypassing encoded motor's SetPower